tileserver-gl.yaml

```diff
@@ -0,0 +1,168 @@
+package:
+  name: tileserver-gl
+  version: 5.0.0
+  epoch: 0
+  description: Vector and raster maps with GL styles. Server side rendering by MapLibre GL Native. Map tile server for MapLibre GL JS, Android, iOS, Leaflet, OpenLayers, GIS via WMTS, etc.
+  copyright:
+    - license: BSD-2-Clause
+  dependencies:
+    runtime:
+      - Xvfb
+      - busybox
+      - mesa
+      - mesa-glx
+      - nodejs-20
+      - npm
+      - xkbcomp
+      - xkeyboard-config
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - cairo-dev
+      - cmake
+      - curl-dev
+      - fribidi-dev
+      - harfbuzz-dev
+      - icu-dev
+      - jq
+      - libglfw-dev
+      - libglvnd-dev
+      - libjpeg-turbo-dev
+      - libpng-dev
+      - libuv-dev
+      - libwebp
+      - libwebp-dev
+      - libxft-dev
+      - mesa
+      - mesa-glx
+      - nodejs-20
+      - npm
+      - pango-dev
+      - pixman-dev
+      - py3.11-pip
+      - python-3.11
+      - xorg-server-dev
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/maptiler/tileserver-gl
+      tag: v${{package.version}}
+      expected-commit: 6cda7a0b384940b781838be7415b2ae448edcd29
+      destination: app
```

Comment:
instead of doing that we have `working-directory:` option where you can use during the pipeline steps, it helps you to avoid using cd app/ in every step, also it makes your build config lot easier to read and follow in my humble opinion, I don't want to be a pedantic here but you can look for other examples which uses working-directory option, it looks a bit lean to me, does that make sense?

Reply:
Thanks. Updated.